x/wasm/types/test_fixtures.go

```diff
@@ -50,7 +50,7 @@ func randBytes(n int) []byte {
 	r := make([]byte, n)
 	rand.Read(r)
 	return r
-}
+}beta2
```

Comment:
I bet that was a typo

Reply:
yep

Reply:
re: does it matter?

honestly when I do stuff sometimes I just choose to be a slave to the linters